CMakeLists.txt

```diff
@@ -58,25 +58,39 @@ set(cefArchiveURL https://cef-builds.spotifycdn.com/${cefName}.tar.bz2)
 
 # fix the url as the version may contain pluses
 string(REGEX REPLACE "\\+" "%2B" cefArchiveURL ${cefArchiveURL})
-set(cefArchive ${CMAKE_CURRENT_SOURCE_DIR}/${cefName}.tar.bz2)
-
-if(NOT EXISTS ${cefArchive})
-    # download cef version
-    message(STATUS "CEF-CMake: Downloading CEF ${cefArchiveURL}")
-    file(DOWNLOAD ${cefArchiveURL} ${cefArchive}
-        SHOW_PROGRESS
-    )
-
-    # ... and extract
-    message(STATUS "CEF-CMake: Extracting ${cefArchive}")
-    execute_process(
-        COMMAND ${CMAKE_COMMAND} -E tar xzf ${cefArchive}
-        WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
-    )
+if(EXISTS ${CEF_BINARY_FOLDER})
```

Comment:
Should this also check if the cmake variable CEF_BINARY_FOLDER is defined and has some value? 

```
if (CEF_BINARY_FOLDER AND EXISTS ${CEF_BINARY_FOLDER})
```

EXISTS says "Behavior is well-defined only for explicit full paths". I think ${CEF_BINARY_FOLDER} might eval to "" if it isn't set.

Reply:
Yup nice catch, I also did a similar fix in the download section